_content/papers/devils-valley-to-omega-point_barrett.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Devil’s Valley to Omega Point: Reflections on the Emergence of a Theme from the Nō"
+authors: 
+  - "T.H. Barrett"
+editor: "Tadeusz Skorupski"
+external_url: "http://www.shin-ibs.edu/documents/bForum/v2/01Barrett.pdf"
```

Comment:
Due to a Chrome security limitation, all `external_url`s that point to a PDF file must use `https` In this case, this site does support https, so just add the little `s` before the `:`  😊